providers/PaperProvider/src/main/java/net/ess3/provider/providers/PaperCommandSender.java

```diff
@@ -1,21 +1,49 @@
 package net.ess3.provider.providers;
 
+import io.papermc.paper.text.PaperComponents;
 import net.kyori.adventure.audience.MessageType;
 import net.kyori.adventure.identity.Identified;
 import net.kyori.adventure.identity.Identity;
 import net.kyori.adventure.text.Component;
 import net.kyori.adventure.text.ComponentLike;
 import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
 import org.bukkit.command.ConsoleCommandSender;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.function.Consumer;
+
 public class PaperCommandSender extends BukkitSenderProvider {
+    private static final boolean SHOULD_MAKE_JMP_HAPPY;
```

Comment:
Not wrong but `FORWARDING_SENDER_AVAILABLE`

Reply:
```suggestion
    private static final boolean FORWARDING_SENDER_AVAILABLE;
```